pcsx2/Frontend/ImGuiManager.cpp

```diff
@@ -114,29 +114,26 @@ void ImGuiManager::WindowResized()
 	const float new_scale = (display ? display->GetWindowScale() : 1.0f);
 
 	ImGui::GetIO().DisplaySize = ImVec2(static_cast<float>(new_width), static_cast<float>(new_height));
-	ImGui::GetIO().DisplayFramebufferScale = ImVec2(new_scale, new_scale);
 
-	UpdateScale();
+	UpdateScale(std::max(new_scale * static_cast<float>(EmuConfig.GS.OsdScale / 100.0), 1.f));
```

Comment:
Any reason specifically behind passing this as a parameter, instead of computing it in `UpdateScale()`? (can read the display scale there).

Otherwise, LGTM

Reply:
Good question.  I have no clue why I did that
Now computed in `UpdateScale()`